### PR TITLE
New minor version 4.30

### PR DESCRIPTION
**Pull request by northwestwitch**
### Added
- Introduced a `reset dismiss variant` verb
- Button to reset all dismissed variants for a case
- Add black border to Chromograph ideograms
- Show ClinVar annotations on variantS page
- Added integration with GENS, copy number visualization tool
- Added a VUS label to the manual classification variant tags
- Add additional information to SNV verification emails
- Tooltips documenting manual annotations from default panels
- Case groups now show bam files from all cases on align view
### Fixed
- Center initial igv view on variant start with SNV/indels
- Don't set initial igv view to negative coordinates
- Display of GQ for SV and STR
- Parsing of AD and related info for STRs
- LoqusDB field in institute settings accepts only existing Loqus instances
- Fix DECIPHER link to work after DECIPHER migrated to GRCh38
- Removed visibility window param from igv.js genes track
- Updated HPO download URL
- Patch HPO download test correctly
- Reference size on STR hover not needed (also wrong)
- Introduced genome build check (allowed values: 37, 38, "37", "38") on case load
- Improve case searching by assignee full name
- Populating the LoqusDB select in institute settings
### Changed
- Cancer variants table header (pop freq etc)
- Only admin users can modify LoqusDB instance in Institute settings
- Style of case synopsis, variants and case comments
- Switched to igv.js 2.7.5
- Do not choke if case is missing research variants when research requested
- Improve create new gene panel form validation
- Make XM- transcripts less visible if they don't overlap with transcript refseq_id in variant page
- Color of gene panels and comments panels on cases and variant pages

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
Checklist before merging/releasing:

### Added
- [ ] Introduced a `reset dismiss variant` verb
- [ ] Button to reset all dismissed variants for a case
- [ ] Add black border to Chromograph ideograms
- [ ] Show ClinVar annotations on variantS page
- [ ] Added integration with GENS, copy number visualization tool
- [ ] Added a VUS label to the manual classification variant tags
- [ ] Add additional information to SNV verification emails
- [ ] Tooltips documenting manual annotations from default panels
- [ ] Case groups now show bam files from all cases on align view
### Fixed
- [ ] Center initial igv view on variant start with SNV/indels
- [ ] Don't set initial igv view to negative coordinates
- [ ] Display of GQ for SV and STR
- [ ] Parsing of AD and related info for STRs
- [ ] LoqusDB field in institute settings accepts only existing Loqus instances
- [ ] Fix DECIPHER link to work after DECIPHER migrated to GRCh38
- [ ] Removed visibility window param from igv.js genes track
- [ ] Updated HPO download URL
- [ ] Patch HPO download test correctly
- [ ] Reference size on STR hover not needed (also wrong)
- [ ] Introduced genome build check (allowed values: 37, 38, "37", "38") on case load
- [ ] Improve case searching by assignee full name
- [ ] Populating the LoqusDB select in institute settings
### Changed
- [ ] Cancer variants table header (pop freq etc)
- [ ] Only admin users can modify LoqusDB instance in Institute settings
- [ ] Style of case synopsis, variants and case comments
- [ ] Switched to igv.js 2.7.5
- [ ] Do not choke if case is missing research variants when research requested
- [ ] Improve create new gene panel form validation
- [ ] Make XM- transcripts less visible if they don't overlap with transcript refseq_id in variant page

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,14 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
...
 ### Added
+### Fixed
+### Changed
+
+## [4.30]
+### Added
 - Introduced a `reset dismiss variant` verb
 - Button to reset all dismissed variants for a case
-- Add black border to ideograms
+- Add black border to Chromograph ideograms
```
Comment (dnil) [inline CHANGELOG.md:16]: 👍

**Review by dnil — APPROVED** [review]
I'll approve as the code is ok, and it is in principle tested, but quite agree we shall tick off your list before actually releasing!

File: CHANGELOG.md (hunk 1 above)
Comment (northwestwitch) [reply CHANGELOG.md:16]: > quite agree we shall tick off your list before actually releasing!

It's on clinical-db stage now. I'll do some checks and then continue tomorrow

Comment (dnil) [reply CHANGELOG.md:16]: It would be nice with re-uploading an up-to-date MIP case, for the CLI side changes like STR AD..

**Comment by northwestwitch** [comment]
OMG what happened to the gene panel? 😆 

image

**Comment by northwestwitch** [comment]
> OMG what happened to the gene panel? 😆

I know that it was me initially to introduce that yellow for the comments, but since already have requests for dark theme why don'y we tone it down a bit?

**Comment by dnil** [comment]
Yeah, agreed. The previous odd/even colors would work fine for comments as well methinks?

**Comment by northwestwitch** [comment]
> Yeah, agreed. The previous odd/even colors would work fine for comments as well methinks?

There was no differential color before for the comments. We could keep the blue and instead of yellow we could have the gray background color perhaps?

**Comment by dnil** [comment]
Sure! Or just the beige background interleaved with gray? Try one and pick one that looks ok with both comments and panels!

**Comment by northwestwitch** [comment]
Mmm looks like that after resetting the dismissed status of the variants (that works) it's not showing that a user (name) has done it, in the case events, but it should because we have created the verb. I'll look into it.

**Comment by northwestwitch** [comment]
> Mmm looks like that after resetting the dismissed status of the variants (that works) it's not showing that a user (name) has done it, in the case events, but it should because we have created the verb. I'll look into it.

Ok that's because the reset dismissed is showed in the variant page, not in the case one. Perhaps it would be better to create an additional event, specific for the case when the reset happens. But we can take in another PR perhaps!

**Comment by dnil** [comment]
> > Mmm looks like that after resetting the dismissed status of the variants (that works) it's not showing that a user (name) has done it, in the case events, but it should because we have created the verb. I'll look into it.
> 
> Ok that's because the reset dismissed is showed in the variant page, not in the case one. Perhaps it would be better to create an additional event, specific for the case when the reset happens. But we can take in another PR perhaps!

Well spotted, but yeah, even if we add one on the case level sometime, we probably still want to keep the variant level ones anyway.

**Comment by northwestwitch** [comment]
Testing the upload of a recent case on hasta stage:

image
image

Loading the case keenviper: 
image

image

**Pull request merged**